### PR TITLE
Lists post

### PR DESCRIPTION
**Pull request by rachelandrew**
An article by an external writer covering design tips for Lists, also adds Michelle as a contributor.

### CODE REVIEW

**Comment by rachelandrew** [comment]
@mbarker84 - take a look, I've made those changes.

**Comment by mbarker84** [comment]
Thanks @rachelandrew, looks good to me!

**Comment by rachelandrew** [comment]
@mbarker84 lovely, I'll publish this next week!

File: src/site/content/en/blog/creative-list-styling/index.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
+---
+title: Creative list styling
+subhead: >
+  A look at some useful and creative ways to style a list.
+description: >
+  A look at some useful and creative ways to style a list.
+date: 2022-08-23
```
Comment (rachelandrew) [inline src/site/content/en/blog/creative-list-styling/index.md:7]:
```suggestion
date: 2022-08-24
```

**Pull request merged**